scripts/install-wp-typia.ps1

```
@@ -0,0 +1,153 @@
+param(
+	[string]$Version = "latest",
+	[string]$InstallDir = "$env:LOCALAPPDATA\wp-typia\bin",
+	[switch]$Update,
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'Update' has been declared but not used.

(PSReviewUnusedParameter)